Firestore/Example/Firestore.xcodeproj/xcshareddata/xcschemes/Firestore-Example.xcscheme

```diff
@@ -49,6 +49,16 @@
                ReferencedContainer = "container:Firestore.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
```

Comment:
It looks like this addition is causing the build failure.

Reply:
I don't understand why that would be.

`Firestore_Tests` and `Firestore_IntegrationTests` are targets that are mentioned immediately above and `Firestore_SwiftTests_iOS` is just another target like those.

Reply:
Okay, the issue here was that I went back and forth on where `Firestore_SwiftTests_iOS` should live in the `Firestore/Example/Podfile`. First I had it nested, then I had it separate and pod updated in both locations.

This worked locally because `DerivedData` still had the output of the previous builds. Once I purged `DerivedData` the local build failed too.

I rebuilt the Xcode project integration without the waffling and can now build cleanly locally, even after purging `DerivedData`. Let's still wait for Travis, of course, but I think this should now be resolved.